src/features/Comment/Comment.jsx

```diff
@@ -26,9 +26,7 @@ export const Comment = ({ permalink }) => {
               alt={`${comment.author} profile `}
               className='user profile image'
```

Comment:
spaces between names are done on purpose ? or you meant user-profile-image  ?  because there it take 3 names class